test/SpecHelper.hs

```diff
@@ -172,6 +177,8 @@ authHeaderJWT :: BS.ByteString -> Header
 authHeaderJWT token =
   (hAuthorization, "Bearer " <> token)
 
+acceptHeader :: BS.ByteString -> Header
```

Comment:
It seems this function is unused. There's also a similar one called `acceptHdrs` above.